src/main/java/org/triplea/client/launch/LaunchScreen.java

```diff
@@ -0,0 +1,654 @@
+package org.triplea.client.launch;
```

Comment:
@RoiEXLab @DanVanAtta This was probably already known to you both, but I learned something new about JFX today related to my previous comment above.

I knew JFX allowed embedding Swing components in a JFX application.  However, I wasn't aware that the opposite was supported: namely, embedding JFX nodes in a Swing application.  This is apparently possible via the [`JFXPanel`](https://docs.oracle.com/javase/8/javafx/api/javafx/embed/swing/JFXPanel.html) class.  As long as all JFX operations are done on the JFX application thread, there should be no issues.

With that said, is there any reason why we couldn't start using JFX for new UI components as they are needed?  That is, as opposed to rewriting an entire JFX UI behind a feature toggle, existing UIs could be incrementally converted to JFX.  Obviously, there will be differences between the Swing L&F and the JFX theme, but as has been mentioned elsewhere, we're possibly about to go into Swing L&F limbo soon, so users may tolerate some visual differences between different parts of the application, especially if they perceive the JFX UI more attractive (which @RoiEXLab's mockups seem to indicate).